library/ssl_tls.c

```diff
@@ -11222,6 +11387,7 @@ static int ssl_write_real( mbedtls_ssl_context *ssl,
 #if defined(MBEDTLS_SSL_PROTO_TLS)
         {
             len = max_len;
+            len_dup = len;
```

Comment:
A very small thing, but if you could do that in the future:
```suggestion
            len = max_len;
            len_dup = max_len;
```
This is more readable (Apologize if I'm too picky :) )

Reply:
This is not about readability, this is about meaning - `len_dup` should always be a clone of `len` - and if someone changes the line above to `len = 10`, no change will be needed for `len_dup`.